chore: windows help baseline (#266)
keymanapp/help.keyman.com: products/windows/14.0/start/tutorial.md

# Getting Started with Keyman

Follow these steps for a basic introduction to Keyman:

<div class="note">

Keyman

must be installed first. For help on installing

Keyman

> , please see:

Reviewer: missing link

.

</div>

> **Step 1**

Reviewer: ```suggestion
## Step 1
```

Check the Windows Taskbar near the clock to see if Keyman is running. If
it is running you will see the Keyman icon:
![](desktop_images/icon-keyman.png)

> **<span id="tutorial_step2"></span>Step 2**

Reviewer: ```suggestion
## <span id="tutorial_step2"></span>Step 2
```

            appear in the Windows Taskbar near the clock, if Keyman is
            on.

        5.  Continue on to [Step 5](#tutorial_step5) of this guide.

    -   On Windows 8:

        1.  Right-click on the Windows Taskbar.

            ![](desktop_images/win8-taskbar1.png)

        2.  Select 'Properties'.

        3.  Next to 'Notification area', click Customize….

            ![](desktop_images/win8-taskbar2.png)

        4.  From the dropdown menu beside SIL International Keyman
            Engine x86, select 'Show icon and notifications'.

            ![](desktop_images/win8-taskbar3.png)

        5.  Click OK to apply changes. The Keyman icon will now always
            appear in the Windows Taskbar near the clock, if Keyman is
            on.

        6.  Continue on to [Step 5](#tutorial_step5) of this guide.

    -   On Windows 10

        1.  Right-click on the Windows Taskbar.

            ![](desktop_images/win10-taskbar1.png)

        2.  Select 'Taskbar settings'.

        3.  Under 'Notification area', click on 'Select which icons
            appear on the taskbar'

            ![](desktop_images/win10-taskbar2.png)

>         4.  From the toggle button beside 'SIL International Keyman
>             Engine x86', click to turn it on.

Reviewer: ```suggestion
        4.  From the toggle button beside 'Keyman Engine x86', click to turn it on.
```

            ![](desktop_images/win10-taskbar3.png)

        5.  The Keyman icon will now always appear in the Windows
            Taskbar near the clock, if Keyman is on.

        6.  Continue on to [Step 5](#tutorial_step5) of this guide.

> **Step 3**

Reviewer: ```suggestion
## Step 3
```

To Start Keyman, click on the Windows Start Menu, select All Programs,
and select Keyman.

> **Step 4**

Reviewer: ```suggestion
## Step 4
```

Keyman will now be running and the Keyman icon will be on the Windows
Taskbar near the clock.

> **<span id="tutorial_step5"></span>Step 5**

Reviewer: ```suggestion
## <span id="tutorial_step5"></span>Step 5
```

Open a typing application, like Microsoft Word.

> **Step 6**

Reviewer: ```suggestion
## Step 6
```

Select a Keyman keyboard from the Keyman menu. You will notice the
Keyman menu icon change to the icon of your Keyman keyboard. This means
your Keyman keyboard is on.

<div class="tip">

If there are no Keyman keyboards in the Keyman menu, you can learn how

> to install one here:

Reviewer: missing link

</div>

> **Step 7**

Reviewer: ```suggestion
## Step 7
```

To learn how to type with your new Keyman keyboard, open the On Screen
Keyboard tool of the Keyman Toolbox. You can open the On Screen Keyboard
tool from the On Screen Keyboard link in the Keyman menu.

<div class="tip">

Some Keyman keyboards do not include an On Screen Keyboard. You can
usually find help for these keyboards from the Keyboard Usage tool of
the Keyman Toolbox. You can open the Keyboard Usage tool from the
Keyboard Usage link in the Keyman menu.

</div>

> **Step 8**

Reviewer: ```suggestion
## Step 8
```

You can now start typing in your language in almost any application. Use
your hardware keyboard or click the On Screen Keyboard keys to type.

> **Step 9**

Reviewer: ```suggestion
## Step 9
```

To return to using your normal keyboard, open the Keyman menu and click

> 'Switch Keyman Off'.

Reviewer: I don't see 'Switch Keyman Off'. Do you mean Exit, or selecting the normal keyboard?

> **Related Topics**

Reviewer: ```suggestion
## Related Topics
```
